uni/ulsp/lsif.icn

```diff
@@ -0,0 +1,392 @@
+                              result2["rangeID"] := rangeID
+                              result2["result"] := result
+                              result2["docID"] := documentID
+                              reference_table[rangeID] := result2
+                           }
+                        } else if \result2 & tojson(result2) ~== "{}" & result2["uri"] ~== uri then {
+                              result2["rangeID"] := rangeID
+                              result2["result"] := result
+                              result2["docID"] := documentID
+                              reference_table[rangeID] := result2
+                        } else {
+                           createHoverResultVertex(result["item"], rangeID)
+                        }
+                        k := result["ends"]
+                     }
+                     k := k + 1
+                  }
+                  j := j + 1
+                  k := 0
+               }
+               if \range_list then {
+                  createContainsEdge(documentID, range_list)
+               }
+            }
+         }
+         if \resultSet_table then {
+            every resultSet := !resultSet_table do {
+               every reference := !reference_table do {
+                  if (resultSet["uri"] == reference["uri"] &&
+                     resultSet["range"]["start"]["line"] == reference["range"]["start"]["line"] &&
+                     resultSet["range"]["end"]["line"] == reference["range"]["end"]["line"] &&
+                     resultSet["range"]["start"]["character"] == reference["range"]["start"]["character"] &&
+                     resultSet["range"]["end"]["character"] == reference["range"]["end"]["character"]) then {
+                     createHoverResultVertex(reference["result"]["item"], reference["rangeID"])
+                     createItemEdge(resultSet["referenceID"], reference["rangeID"], reference["docID"], "references")
+                     delete(reference_table, reference["rangeID"])
+                     delete(resultSet_table, resultSet["resultSetID"])
+                  }
+               }
+            }
+         }
+         if \document_list then {
+            every documentID := !document_list do {
+               createEventVertex("document", "end", documentID)
+            }
+            createContainsEdge(projectID, document_list)
+         }
+         createEventVertex("project", "end", projectID)
+      }
+      close(outFile)
+   end
+
+   #  Iterates nextID up one.
+   #  <i>This is intended for internal use only!</i>
+   method getNextID()
+      return nextID +:= 1
+   end
+
+   #  Creates a metaData vertex with the given information and outputs it to the output file.
+   #  Params: path - The path of the main Unicon directory.
+   #  <i>This is intended for internal use only!</i>
+   method createMetaDataVertex(path)
+      local lsif_table
+
+      lsif_table := table()
+      lsif_table["id"] := nextID
```

Comment:
Why not call `getNextID()` here, and drop line 164? same for all similar uses in other functions.